packages/serverless/src/domain/use-case/merkle-tree.ts

```diff
@@ -1,6 +1,6 @@
 import { ModelMerkleTree } from '@zkdb/storage';
 import { ClientSession } from 'mongodb';
-import ModelDocumentMetadata from '../../model/database/metadata-document.js';
+import ModelDocumentMetadata, { ModelMetadataDocument } from '../../model/database/metadata-document.js';
```

Comment:
This one is a little bit confuse, can we only choose one export instead of default export.